pkgs/applications/editors/vim/plugins/vim-utils.nix

```diff
@@ -201,7 +201,7 @@ let
         ln -s ${python3Env}/${python3Env.sitePackages} $out/pack/${packageName}/start/__python3_dependencies/python3
       '';
     in
-      [ packdirStart packdirOpt python3link ];
+      [ packdirStart packdirOpt ] ++ lib.optional (allPython3Dependencies {} != []) python3link;
```

Comment:
This passes `{}` as `ps`, which may break evaluation if someone writes
`python3Dependencies = ps: lib.optionals (ps.somepkg.somecondition) [ ps.someotherpkg ];`
We doesn't have this pattern in nixpkgs currently. But just to remind.

Reply:
That's a good point, I guess we should pass `python3.packages` instead?